libs/datamodel/core/src/ast/reformat/reformatter.rs

```diff
@@ -364,7 +364,53 @@ impl<'a> Reformatter<'a> {
 
     fn reformat_field(&self, target: &mut TableFormat, token: &Token, model_name: &str) {
         let field_name = &Self::get_identifier(token);
-        for current in token.clone().into_inner() {
+
+        // get indices of directives and store in separate Vector
+        let mut directive_indices = Vec::new();
+        let mut directives = Vec::new();
+        for (i, pair) in token.clone().into_inner().enumerate() {
+            match pair.as_rule() {
+                Rule::directive => {
+                    directive_indices.push(i);
+                    directives.push(pair);
+                }
+                _ => (),
+            }
+        }
+
+        // sort directives
+        let correct_order = vec!["map", "id", "unique", "default", "relation", "updatedAt"];
+        directives.sort_by(|a, b| {
+            let mut sort_index_a = usize::MAX;
+            let mut sort_index_b = usize::MAX;
+            println!("{:?}", sort_index_a);
```

Comment:
maybe use the logger here instead? Something like `debug!` or `trace!`, so we don't spam the STDOUT.